src/main.ts

```diff
@@ -13,11 +16,32 @@ registerPlugins(app)
 registerUtils(app)
 
 
+axios.defaults.withCredentials = true
 
 // Globals
 app.config.globalProperties.$_ = _
 app.config.globalProperties.$isInIframe = !!window.parent.frames.length
 
 const apiBaseUrl = import.meta.env.VITE_APP_API_BASE_URL
 
-app.mount('#app')
+axios.get(`${apiBaseUrl}/api/user/profile`).then(response => {
+  const data = response.data
+  useContextStore().setCurrentUser(data)
+
+  axios.get(`${apiBaseUrl}/api/config`).then(response => {
+    const data = response.data
+    useContextStore().setConfig({
+      ...data,
+      apiBaseUrl,
+      isVueAppDebugMode: _.trim(import.meta.env.VITE_APP_DEBUG).toLowerCase() === 'true'
+    })
+    app.use(router).config.errorHandler = function (error, vm, info) {
+      const message = _.get(error, 'message') || info
+      const stacktrace = _.get(error, 'stack', null)
+      console.log(`\n${message}\n${stacktrace}\n`)
+      useContextStore().setApplicationState(500, message, stacktrace)
+    }
+    app.mount('#app')
+  })
+})
```

Comment:
This is a more modern replacement for the removed `initSession` code above. It pulls config values and current user profile from the API and stashes them in the front-end context store.